rs/nns/governance/BUILD.bazel

```diff
@@ -337,28 +332,37 @@ rust_bench(
     ],
 )
 
+# all TLA modules
+# NOTE: the test runner assumes unique basenames
+filegroup(
+    name = "tla_modules",
```

Comment:
Let's keep "models" instead of "modules" here. While modules is technically also correct in the sense that TLA has a module system, to people who don't know  TLA well I think "models" is more descriptive.